enhancements/api-review/custom-tags-aws.md

```diff
@@ -91,19 +151,23 @@ AWS resource tags.
 
 ### Test Plan
 
-TBD
+Update the userTag testcases to check successful update from infrastructure resource.
 
 ### Graduation Criteria
 
-TBD
-
 #### Dev Preview -> Tech Preview
```

Comment:
it's already TP, so I think you can leave this as "NA" or something like that

Reply:
Ok, done.